### PR TITLE
Add Logback config for KMS tests

### PR DESCRIPTION
**Pull request by meltsufin**
Avoid too verbose `DEBUG` level output.

### CODE REVIEW

File: spring-cloud-gcp-kms/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<!-- encoders are assigned the type
```
Comment (elefeint) [inline spring-cloud-gcp-kms/src/test/resources/logback-test.xml:4]: Remove comment? This is not relying on default.

File: spring-cloud-gcp-samples/spring-cloud-gcp-kms-sample/src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<!-- encoders are assigned the type
```
Comment (elefeint) [inline spring-cloud-gcp-samples/spring-cloud-gcp-kms-sample/src/test/resources/logback-test.xml:4]: Same.

**Review by elefeint — APPROVED** [review]
I wonder if we had a test-support project if we could just have one place for the test logback config. Test-support came up before, I think, in the context of emulators.

File: spring-cloud-gcp-kms/src/test/resources/logback-test.xml (hunk 1 above)
Comment (meltsufin) [reply spring-cloud-gcp-kms/src/test/resources/logback-test.xml:4]: Why do you say it's not relying on default? When you use `<encoder>` without specifying the class, it's supposed to be the `PatternLayoutEncoder`.

Comment (elefeint) [reply spring-cloud-gcp-kms/src/test/resources/logback-test.xml:4]: I meant that there is an explicit `<pattern>` tag, so it's obvious that `PatternLayoutEncoder` is used even without the comment.

**Pull request merged**